scripts/categorizer.py

```diff
@@ -378,6 +383,9 @@ def main():
     hash_storage = {}
 
     LineAndCats = collections.namedtuple('LineAndCats', 'line multiline_statement_number comment conditional empty func_def assignment func_call')
+    # https://stackoverflow.com/questions/11351032/namedtuple-and-default-values-for-optional-keyword-arguments
+    LineAndCats.__new__.__defaults__ = (0,) * len(LineAndCats._fields)
```

Comment:
Is all of this code in Python 2.7? That link you provided (which I saw I've upvoted a while back!) provides an elegant solution with Python 3.7+

Reply:
I am currently using python 3.5 currently with a virtual environment. I need to add this to the readme.